projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-content/user-contributions/user-contributions.component.scss

```diff
@@ -1,43 +1,62 @@
 @import '/projects/commudle-admin/src/assets/styles/variables';
 
 .user-contributions {
-  .header {
-    margin: $space-40 0 $space-24;
-    font-size: $fs-24;
-    font-weight: $fw-600;
-    line-height: 28px;
-    color: $neon-blue;
-  }
-
-  .content {
-    padding-bottom: $space-16;
-  }
+  .section {
+    position: relative;
 
-  .user-contributions-builds {
-    .user-contributions-builds-cards {
-      overflow: auto;
-      gap: $space-32;
+    .header {
+      margin: $space-40 0 $space-24;
+      font-size: $fs-24;
+      font-weight: $fw-600;
+      line-height: 28px;
+      color: $neon-blue;
     }
-  }
 
-  .user-contributions-labs {
-    .user-contributions-labs-cards {
-      overflow: auto;
+    .content {
+      display: flex;
```

Comment:
should we move this to classes?

Reply:
There are a lot of content classes and I'm selecting them as a whole, that's why I put the display in scss file so we can control them from a single place